src/ip-ranges.md

```diff
@@ -2,6 +2,15 @@
 title: 'IP Ranges'
 ---
 
+{% capture __alert_content -%}
+The contents of this page only apply to Sentry's SaaS product. It **does not** apply to on-premise or single tenant.
```

Comment:
@dcramer do we officially refer to `single tenant` as `single-tenant` or?

Reply:
@dcramer just checkin' in on this